nebula_messages/continental_msgs/msg/ContinentalArs548Object.msg

```diff
@@ -0,0 +1,44 @@
+uint32 object_id
+uint16 age
+uint8 status_measurement
+uint8 status_movement
+uint8 position_reference
+
+geometry_msgs/Point position
+geometry_msgs/Vector3 position_std
+float32 position_covariance_xy
+float32 position_orientation
+float32 position_orientation_std
```

Comment:
`(optional)` The name in the datasheet is confusing, just orientation would be clearer.

Reply:
Addressed in 
d383931